### ISSUE TITLE
T2 L6/9 - Different representations of state vector in video and text

### ISSUE THREAD

**Issue by sahiljuneja** [udacity/sdc-issue-reports#535]
Not sure how important this is, but in the video we have this - 

And in the text below that, we have this - 

Not sure if the "prime" is important or not here, but thought of pointing this out to avoid confusion.

**Comment by andrewpaster** [comment]
Yeah, this is a little confusing. The main point of this lecture is to show how the H matrix drops vx and vy from the state vector. Then the state vector has the same dimensions as the lidar measurement and can be compared. 

The prime just refers to the fact that you've already done the update step but haven't done the measurement step yet. 

Either all values px, py, vx and vy on both sides of the equation should have the prime or none of them should have the prime. Which way do you think would be clearer? (I'm leaning towards no prime to keep the focus on what the H matrix does).

**Comment by sahiljuneja** [comment]
Honestly, whichever is easier for you guys. Don't know how easy it is for you to edit videos. 

But you point out this above 
```The prime just refers to the fact that you've already done the update step but haven't done the measurement step yet.```

If you could add that to the explanation [assuming there isn't any right now, it's been a while since I saw it] and match the text to the video, then I don't think there will be any confusion at all. What do you think?

**Comment by andrewpaster** [comment]
Okay, I made some updates to the classroom. Thanks for pointing this out and getting back to me about it. Hope it clarifies things for future students!
